MapboxNavigation/NavigationMapView.swift

```diff
@@ -29,7 +29,7 @@ open class NavigationMapView: MGLMapView, UIGestureRecognizerDelegate {
     /**
      Returns the altitude that the map camera initally defaults to.
      */
-    public var defaultAltitude: CLLocationDistance = 1000.0
+    public var defaultAltitude: CLLocationDistance = 850.0
```

Comment:
Should we make the map generally zoom in closer during urban driving, as this change does, or should we distinguish between the default altitude and the altitude as the user approaches the destination, also making that altitude configurable? It would be pretty straightforward to make an additional property for that.